src/main/java/org/bahmni/sms/impl/DefaultSmsSender.java

```diff
@@ -35,12 +34,17 @@ public ResponseEntity send(String phoneNumber, String messageText) {
         logger.info("Sending SMS for ********" + phoneNumber.substring(phoneNumber.length() - 2));
         try {
             HttpClient httpClient = HttpClientBuilder.create().build();
-            HttpPost request = new HttpPost(SEND);
+            HttpPost request = new HttpPost(smsProperties.getProviderApi());
             Message message = new Message();
             message.setChannel("sms");
             message.setMsg_type("text");
+
+            if ((phoneNumber.length()==10 || phoneNumber.length()==11) && !phoneNumber.startsWith("+")) {
```

Comment:
Why is the length based check required here ? Will this work for different country phone numbers ?